worker-pools.yml

```diff
@@ -1666,6 +1666,30 @@ pools:
             - <<: *persistent-disk
               diskSizeGb: 60
           machine_type: c2-standard-4
+  - pool_id: '{pool-group}/t-linux-2404-headless-arm64-alpha'
+    description: Workers running Ubuntu 24.04 arm64. Used by RelSRE to qualify images.
+    owner: release+tc-workers@mozilla.com
+    email_on_error: true
+    provider_id: fxci-level1-gcp
+    variants:
+      - pool-group: gecko-t
+    config:
+      worker-config:
+        genericWorker:
+          config:
+            enableInteractive: true
+            headlessTasks: true
+      minCapacity: 0
+      maxCapacity: 100
+      implementation: generic-worker/linux-d2g
```

Comment:
I assume you intentionally enabled d2g here? (I'm not sure what this pool will be used for, so I'm not sure whether it's correct.)

Reply:
@bhearsum I am going to run some test harnesses for the ARM64 builds on these. We are piggy backing on the x64 image that will run d2g

Reply:
I'll defer to you on whether or not this is correct, in that case!